### DISCUSSION TITLE
Is there a way to get the compile options used from a device running?

### DISCUSSION THREAD

**Discussion by Kugeleis** [arendst/Tasmota#21553] [Q&A]
I compiled a version with some options with the online tool [tasmocompiler ](https://github.com/benzino77/tasmocompiler) quite a while ago. I do not remember the compile options used. Is there a way to retrieve them from the device?

**Comment by sfromis** [comment]
Not all of them, but many can be seen via tools
https://github.com/tasmota/decode-config/
https://github.com/tasmota/decode-status/
